ci/pipelines/toolbox-deploy-ecr.yml

```
@@ -0,0 +1,102 @@
+resources:
+  - name: toolbox-deploy-ecr
+    type: git
+    icon: github
+    source:
+      uri: https://github.com/alphagov/pay-ci
+      branch: master 
+      paths:
+        - ci/pipelines/toolbox-deploy-ecr.yml
+  # Github Releases
+  - name: git-toolbox-staging-release
+    type: git
+    icon: github
+    source: &github-release-source
```

```suggestion
    source:
```
This YAML anchor is not used. Do we need it?